precompiles/avs/IAVSManager.sol

```diff
@@ -158,10 +186,11 @@ interface IAVSManager {
     ) external view returns (uint256 amount);
 
     /// @dev RegisterBLSPublicKey Emitted when `operator` registers with the public keys `pubKey`.
-    /// @param operator the address of the delegator
-    /// @param pubKey the address of the validator
+    /// @param name the name of the blsKey
+    /// @param pubKey the public key of the operator
+    /// @param pubkeyRegistrationSignature the sig of the operator
+    /// @param pubkeyRegistrationMessageHash the sig hash of the operator
     event RegisterBLSPublicKey(
```

Comment:
For clarity, it is better to rename the event to something like `BLSPublicKeyRegistered`, since it currently sounds the same as the function name `registerBLSPublicKey`. Also, please check if all the events defined in this contract interface are being emitted in the respective locations; this was marked as an informational item in the audit.

This may be done in another PR; it should not block this one.